src/.vuepress/theme/layouts/Layout.vue

```diff
@@ -192,10 +216,26 @@ export default {
 
     toggleBannerEvents(on) {
       // Add or remove event listerners attached to the DOM
-      let method = on ? "addEventListener" : "removeEventListener"
-      window[method]("resize", this.getMenuPosition)
-      window[method]("scroll", this.fixMenuAfterBanner)
+      let method = on ? 'addEventListener' : 'removeEventListener'
+      window[method]('resize', this.getMenuPosition)
+      window[method]('scroll', this.fixMenuAfterBanner)
     }
   }
 }
 </script>
+
+<style>
+/* add a placeholder to give space to ads so they don't overlap with content */
+@media screen and (max-width: 1376px) {
+  .content__default::before {
```

Comment:
This to make the ad take some space

vuejs.org, ad inside content

![Screen Shot 2020-12-02 at 13 25 04](https://user-images.githubusercontent.com/664177/100872305-e87fca00-34a1-11eb-8ca2-d0d9fd213aaa.png)

vuepress doesn't:

![Screen Shot 2020-12-02 at 13 25 11](https://user-images.githubusercontent.com/664177/100872331-f2a1c880-34a1-11eb-9921-b56fccd2624c.png)


But it creates overlap for tip containers unless we add `overflow-x: auto` but that breaks the icon display:

![Screen Shot 2020-12-02 at 13 20 02](https://user-images.githubusercontent.com/664177/100872384-064d2f00-34a2-11eb-8993-5de23bb48207.png)

Which is the reason I added the background